free-1.md

```diff
@@ -5,47 +5,108 @@
 
+    I think a DSL is appropriate because calendars sometimes feel too cluttered?
+    This might be a me problem, but when making an event on google calendar I
+    have 8 options for how it repeats, the option to add notes, google meet links,
+    a visibility option, location options, guest permissions, and I don't need all
+    of that I just need it to repeat weekly and have a start/end time and name.
+    It would be nice if there were a text-based DSL that could quickly generate
+    calendars by taking in some input, which could then be automated by Python or
+    some other external language. 
+
 ### Why you?
 
 _What excites you about this idea? How did you come up with it?_
 
+    I had the idea to make some alternative scheduling/calendar solution over break
+    and toyed with the google calendar API in a frontend for a bit.
+    It felt like Google calendar was both too broad in it's capabilities, but was still
+    so frustrating at times. For instance, if I wanted to add an event to my calendar
+    but didn't want to attend, google would mark it as having attended/spent time there.
+    I have the dining hall hours on my google calendar and it says I spend over 50% of my
+    time in meetings.
+    Basically I like the idea of being able to input in some brief text, and having a full
+    schedule be output.
+
 ### Domain
 
 _Describe the project's domain in five words._
 
+    Weekly schedule creation via text.
+
 ### Interface (syntax)
 
 _How might the user interact with the language? What does programming look
 like? Why is this the right way to interact with the problem domain?_
 
+    The user would interact with the language via a command line or via a text file, with a
+    string of functions called in order to generate the schedule. A preview of the schedule
+    could then be seen while editing, similar to ContextFree.
+    The reason this is the right way to interact with the problem domain is because it is
+    both faster than editing a normal calendar/schedule, and also easier to type it out.
+
 ### Operation (semantics)
 
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+    When a program runs it outputs a generated schedule as a pdf or an image or
+    a csv file. Errors that occur could be type mismatching/incorrect syntax.
+    In these cases a small error message could be shown when using a commandline
+    or from the program being used to see previews. 
+
 ### Expressiveness
 
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+    It should be easy to create events with a start day, start time and end time,
+    name, and when it repeats using text input. It should be very difficult to 
+    add more complexity, and impossible to do much of the functionality that
+    typical calendar apps allow for.
+
```

Comment:
So would this generate a "calendar" file type? (like ics I think) and be editable later, or is it a single output.